### PR TITLE
Setup for now v1

### PR DESCRIPTION
**Pull request by orballo**
fixes https://github.com/frontity/services/issues/10

### CODE REVIEW

**Comment by orballo** [comment]
@luisherranz ready for review :smile:

File: now.json [hunk 1]
```diff
@@ -1,23 +1,9 @@
 {
   "name": "frontity-services",
-  "version": 2,
+  "version": 1,
   "alias": "services.frontity.cloud",
   "regions": ["all"],
```
Comment (luisherranz) [inline now.json]: Can you remove `regions` and add `scale` here as well? Same settings.

**Review by luisherranz — CHANGES_REQUESTED** [review]

File: services/add-media-ids/now.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "version": 1,
+  "alias": "frontity-add-media-ids",
```
Comment (luisherranz) [inline services/add-media-ids/now.json:3]: Don't you need to add `.now.sh` to the alias?

Comment (orballo) [reply services/add-media-ids/now.json:3]: Nopes

File: now.json (hunk 1 above)
Comment (orballo) [reply now.json]: Oh yes, I missed that.

File: services/add-media-ids/now.json (hunk 2 above)
Comment (luisherranz) [reply services/add-media-ids/now.json:3]: Oki doki

**Comment by orballo** [comment]
@luisherranz ready for review 😄

**Review by luisherranz — APPROVED** [review]
Perfect, thanks @orballo.

**Pull request merged**